ui/components/app/collectibles-items/index.scss

```diff
@@ -0,0 +1,18 @@
+.collectibles-items {
+  &__image {
+    border-radius: 0.625rem;
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.collection-icon {
+  border-radius: 50%;
+  width: 2rem;
+  height: 2rem;
+  padding: 0.5rem;
+  background: $ui-4;
+  color: $ui-white;
+  text-align: center;
+  line-height: 1;
+}
```

Comment:
Above styles should be nested I think.

Reply:
Agreed! Thanks! Will address in subsequent PR!